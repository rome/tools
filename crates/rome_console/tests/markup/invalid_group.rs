fn main() {
    rome_console::markup! {
        []
    }
}
