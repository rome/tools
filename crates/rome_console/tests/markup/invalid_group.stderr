error: unexpected token
 --> tests/markup/invalid_group.rs:3:9
  |
3 |         []
  |         ^^
